node/test/max_pending.js

```diff
@@ -28,7 +28,7 @@ var allocCluster = require('./lib/alloc-cluster');
 testBattle('cap maximum pending requests', {
     numPeers: 2,
     channelOptions: {
-        maxPending: 1,
+        maxPending: 2,
```

Comment:
Actually, it was working on the edge. After we lift the max pending to TChannelRequest.send(), max pending checks have a higher priority than timeout checks. So, we need to update the test.